### ISSUE TITLE
ConstructorError: could not determine a constructor for the tag '!Equals'

### ISSUE THREAD

**Issue by hawry** [hawry/deforest#4]
**Describe the bug**
Running deforest on CF containing non-supported shorthand AWS functions fails.

**Expected behavior**
Ignore unknown AWS functions and continue with the export.

**Screenshots or error logs**
```
yaml.constructor.ConstructorError: could not determine a constructor for the tag '!Equals'
  in "<string>", line 38, column 21:
      ConditonalLogicalId: !Equals  #If site equals  ...
```

**Desktop (please complete the following information):**
WSL on Win 10

**OAS/Swagger version:**
OAS 3

**Additional context**
CloudFormation versino 2010-09-09

**Comment by hawry** [comment]
Should be fixed in #6 , waiting for confirmation before closing

**Comment by hawry** [comment]
Fixed in 5416f4e9
